documentation/docs/03-loading.md

```diff
@@ -62,6 +62,7 @@ Our example blog page might contain a `load` function like the following. Note t
 	}
 </script>
 ```
+> Note the `<script context="module">` — this is necessary because `load` runs before the component is rendered. Your "regular" code should go into a second `<script>` tag.  
```

Comment:
```suggestion
> Note the `<script context="module">` — this is necessary because `load` runs before the component is rendered. Code that is per-component instance should go into a second `<script>` tag.  
```